package.json

```diff
@@ -105,6 +105,7 @@
     "tippy.js": "^6.3.5",
     "ttag": "1.7.15",
     "underscore": "~1.13.3",
+    "workbox-webpack-plugin": "^7.0.0",
```

Comment:
Probably need to add `workbox-strategies` and `workbox-routing` here, since we use them directly in sw.js

Reply:
They're indirect dependencies and got installed and listed in the yarn.lock file automatically (and I'd hope to get an error if something I import is missing? Asking for real, Webpack and Workbox looks to me a bit like Asterix's "La maison qui rend fou"...)

Reply:
They're not indirect since we use them in `sw.js`